qiskit/circuit/quantumcircuit.py

```diff
@@ -4880,6 +5545,10 @@ def qubit_stop_time(self, *qubits: Union[Qubit, int]) -> float:
         return 0  # If there are no instructions over bits
 
 
+# isometry is an alias for iso
+QuantumCircuit.isometry = QuantumCircuit.iso
```

Comment:
Not important at all, just saying: it might have been a shade cleaner to do this within the definition of `QuantumCircuit` -
```python
class QuantumCircuit:
    def iso(self, ...): ...

    # isometry is an alias for iso
    isometry = iso
```